core/arch/x86/decode_table.c

```diff
@@ -2783,6 +2807,10 @@ const instr_info_t prefix_extensions[][8] = {
     {OP_vmovsldup,0xf30f1210, "vmovsldup", Vvs, xx, Wvs, xx, xx, mrm|vex, x, END_LIST},
     {OP_vmovlpd,  0x660f1210, "vmovlpd", Vq_dq, xx, Hq_dq, Mq, xx, mrm|vex, x, tpe[3][6]},
     {OP_vmovddup, 0xf20f1210, "vmovddup", Vvd, xx, Wvq_dq, xx, xx, mrm|vex, x, END_LIST},
+    {INVALID, 0x0f1210, "vmovlps", xx, xx, xx, xx, xx, no, x, NA},
```

Comment:
INVALID?

Reply:
Yes, because  not implemented yet. Names was written only for helping purpose.

Reply:
@hgreving2304 please resolve discussion if I have answered your question.

Reply:
As long as it is INVALID, it need to be name "(bad)". Any _valid_ instruction needs to be accompanied by a INSTR_CREATE test in ir_x86.c and preferably by another decoder test (there will be a suite going upstream for this soon).